7.10.2/debian-10/rootfs/opt/bitnami/scripts/libkibana.sh

```diff
@@ -181,8 +181,9 @@ is_kibana_not_running() {
 #   Boolean
 #########################
 is_kibana_ready() {
+    local basePath=$(yq r $KIBANA_CONF_FILE "[server.basePath]")
```

Comment:
I think we should create a `kibana_conf_get` function (very similar to the `kibana_conf_set` we already have) and use it here:
```suggestion
    local basePath
    basePath="$(kibana_conf_get "server.basePath")"
```